### ISSUE TITLE
Caption positions in publication

### ISSUE THREAD

**Issue by RisingNeon** [tudace/tuda_latex_templates#73]
Hi, 
While writing in different documents (publication -> class: report; thesis ->class: report) the caption were set on the left side under figures and tables. I'm unsure whether it should be like this or not, because normally its set centered under the figure.
I only tested it in the publication document type and with lualatex compilation.

Kind regards

**Comment by TeXhackse** [comment]
We discussed that some time ago.

KOMA-Script in default setting makes a difference between oneline and multiline captions.
The online-ones are centered, the multiline ones aren't.
Since this difference might be confusing we decided to set the KOMA-default option `captions=nooneline`. 

If you only have single-line captions you might want to change that using the KOMA-Script option directly as an option to the TUDa-CI-class or `\KOMAoptions`. Just add 
```
captions=oneline
```
See the `captions` option in the KOMA-Script documentation for more details on that.
